packages/Overlay/src/index.js

```diff
@@ -0,0 +1,3 @@
+export { default } from "./Overlay";
+export { default as Backdrop } from "./Backdrop";
+export { default as Wrapper } from "./Wrapper";
```

Comment:
Who needs to consume these styled-components besides `<Overlay>`?

Reply:
I decided to expose this components to have ability to style them when we consume them with `styled-components`.

Example:
```jsx
import Overlay, { Backdrop } from '@paprika/overlay';

const StyledOverlay = styled(Overlay)`
  // some styles for overlay

  ${Backdrop} {
    // some styles for backdrop
  }
`;
```

If we remove `Backdrop` from exports, than only way to style backdrop will be:
```jsx
import Overlay from '@paprika/overlay';

let StyledOverlay = styled(Overlay)`
  // some styles for overlay
`;

StyledOverlay = styled(({ className, ...rest }) => <StyledOverlay backdropClassName={className} {...rest} />)`
  // some styles for backdrop
`;
```

I agree that probably exporting `Wrapper` is a bit unnecessary. I can remove it, but think it does not affect anything.